ocaml/xapi-storage/generator/lib/control.ml

```diff
@@ -28,6 +28,8 @@ type health =
   | Healthy of string  (** Storage is fully available *)
   | Recovering of string
       (** Storage is busy recovering, e.g. rebuilding mirrors *)
+  | Unreachable of string  (** Storage is unreachable *)
```

Comment:
would be a good idea to explain what unreachable means, i.e. temporarily out of action, IIRC, and same for unavailable

Reply:
It's sort of complicated. In the case of GFS2 it is raised when the filesystem is not withdrawn (that would be unavailable) but is otherwise not able to satisfy IO requests within a timed, 2 seconds, check. Probably unlikely to happen as it's likely to be the case that either the FS is in the process of withdrawing or the host is about to fence.

Reply:
yes, that's fine, I think a general explanation of the difference between the two would suffice, e.g. why having two states of health instead of one?

Reply:
Done